daffodil-runtime1/src/main/scala/org/apache/daffodil/debugger/InteractiveDebugger.scala

```diff
@@ -1320,21 +1337,40 @@ class InteractiveDebugger(runner: InteractiveDebuggerRunner, eCompilers: Express
           }
           DebugState.Pause
         }
+
+        def diff(prestate: StateForDebugger, state: ParseOrUnparseState): Boolean = {
+          if (prestate.bitLimit0b != state.bitLimit0b) {
+            debugPrintln("%s: %d -> %d".format(name, prestate.bitLimit0b, state.bitLimit0b), "  ")
```

Comment:
codecov says this is uncovered. Probably should have a test that outputs ALL diffs.